packages/arb-token-bridge-ui/src/components/common/DepositCountdown.tsx

```diff
@@ -15,21 +16,29 @@ function getMinutesRemainingText(minutesRemaining: number): string {
 }
 
 export function DepositCountdown({
-  createdAt,
-  depositStatus
+  tx
 }: {
-  createdAt: DayJSConfigType
-  depositStatus?: DepositStatus
+  tx: MergedTransaction
 }): JSX.Element | null {
   const now = dayjs()
+  const createdAt = tx.createdAt
+  const depositStatus = tx.depositStatus
   const whenCreated = dayjs(createdAt)
 
+  // check the transaction belongs to which network, and on basis of that show the deposit timer
+  const { parentChainId } = tx
+  let timerMinutes = 15 // default to 15 mins
+
+  if (parentChainId && !isNetwork(parentChainId).isMainnet) {
```

Comment:
Defaulting everything else to 1 min should do for now (until we have Orbit mainnets). But Goerli txs are going to take longer than a minute I suppose? Might want to cover Ethereum testnet case?
```suggestion
  let timerMinutes = 1 // default to 1 min
  
  if (parentChainId) {
     const { isEthereum, isTestnet } = isNetwork(parentChainId)
     
     if (isEthereum) {
        if (isTestnet) {
            timerMinutes = 5 // or something
        } else {
            timerMinutes = 15
        }
     }
  }
```

Reply:
Hmm I kept it like this on purpose because IF something goes wrong (maybe misconfiguration / localstorage issue or a missed use-case down the line) and we don't have a `parentChainId` present here, it would be better to fallback to the production environment's config of 15mins rather than showing a testnet configuration of 1 minute in production.

Reply:
And about the slab of timers for different types of testnets, yeah that can be done. It's a business logic call which we'll need to confirm with the team.